### PR TITLE
[Modal] Fix shadow

### PR DESCRIPTION
**Pull request by kyledurand**
Modal has a much broader shadow in figma, it's also darker. I've added a token and modified the modal shadow to match [Figma](https://www.figma.com/file/4dAAt5iFPSaxUKiYVKrkYj/DL%E2%80%93Polaris-(Web)%3A-Components?node-id=358%3A0)

Before:
image

After:
image

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}

```

</details>

### 🎩 checklist

* [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
* [ ] Updated the component's `README.md` with documentation changes
* [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

File: src/utilities/theme/tokens.ts [hunk 1]
```diff
@@ -4,12 +4,13 @@ const BorderRadius = {
 };
 
 const Shadow = {
+  shadowFromDimLight: 'rgba(0, 0, 0, 0.2)',
```
Comment (danrosenthal) [inline src/utilities/theme/tokens.ts]: You'll want to add this to tokens instead: https://github.com/Shopify/polaris-tokens/blob/next/formats/utils/color-factory/configs/base.js#L98

Comment (kyledurand) [reply src/utilities/theme/tokens.ts]: Can we get into the habit of adding these locally until design reviews are completed and then moving them to tokens?

It just seems like a round about way of doing things. I'd have to add the token in the package, bump the version and then make the change in Polaris.

I'd prefer adding locally and then removing chunks later on as they become available in tokens

Comment (danrosenthal) [reply src/utilities/theme/tokens.ts]: I'm comfortable with that. But it should have an accompanying issue for the removal here, and a pull request in tokens.

Comment (kyledurand) [reply src/utilities/theme/tokens.ts]: I'll put the PR up in tokens. Shouldn't we have a PR to move a bunch of local tokens? Or is this the only one you think worth moving right now?

Comment (danrosenthal) [reply src/utilities/theme/tokens.ts]: Currently the only tokens that live in polaris tokens are those generated by the color factory, so no, there shouldn't be anything else we need to migrate currently from `tokens.ts`

Comment (danrosenthal) [reply src/utilities/theme/tokens.ts]: As we start to explore customizable tokens for font-size, border-radius, we'll work to move those tokens over, but that work is not ready yet.

**Pull request merged**